presto-main/src/main/java/com/facebook/presto/operator/HashBuilderOperator.java

```diff
@@ -673,15 +677,17 @@ public void close()
         }
         // close() can be called in any state, due for example to query failure, and must clean resource up unconditionally
         lookupSourceSupplier = null;
+        localRevocableMemoryContext.setBytes(0);
+        localUserMemoryContext.setBytes(index.getEstimatedSize().toBytes(), enforceBroadcastMemoryLimit);
```

Comment:
Why we still count index size, if it's set to null?

Reply:
good catch. I copy-pasted the code from other places and forgot to update it to 0.